tools/xsetwacom.c

```diff
@@ -2934,14 +2983,14 @@ int main (int argc, char **argv)
 	}
 
 	if (do_list)
-		list(dpy, argc - optind, &argv[optind]);
+		status = list(dpy, argc - optind, &argv[optind]);
 	else if (do_set)
-		set(dpy, argc - optind, &argv[optind]);
+		status = set(dpy, argc - optind, &argv[optind]);
 	else if (do_get)
-		get(dpy, format, argc - optind, &argv[optind]);
+		status = get(dpy, format, argc - optind, &argv[optind]);
 
 	XCloseDisplay(dpy);
-	return 0;
+	return status == 0 ? 0 : 1;
```

Comment:
```suggestion
	return status;
```
because status is now set to one of the exit codes, right?

Reply:
Yes now that it can be any of the exit codes putting it back to returning just the status should be proper.